docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+      "shareMeetingChatHistoryDefault": "none",
+      "allowTeamworkReactions": true,
+      "allowAttendeeToEnableMic": true,
+      "allowAttendeeToEnableCamera": true,
+      "recordAutomatically": false,
+      "anonymizeIdentityForRoles": [],
+      "capabilities": [],
+      "videoTeleconferenceId": null,
+      "externalId": null,
+      "iCalUid": null,
+      "meetingType": null,
+      "allowParticipantsToChangeName": false,
+      "allowRecording": true,
+      "allowTranscription": true,
+      "meetingMigrationMode": null,
+      "broadcastSettings": null,
+      "audioConferencing": null,
+      "meetingInfo": null,
+      "participants": {
+        "organizer": {
+          "upn": "john.doe@contoso.com",
+          "role": "presenter",
+          "identity": {
+            "application": null,
+            "device": null,
+            "user": {
+              "id": "b2091e18-7882-4efe-b7d1-90703f5a5c65",
+              "displayName": null,
+              "tenantId": "ad4f158a-97c7-4914-a9bd-038ecde40ff3",
+              "identityProvider": "AAD"
+            }
+          }
+        },
+        "attendees": [
+          {
+            "upn": "adele.vance@contoso.com",
+            "role": "attendee",
+            "identity": {
+              "application": null,
+              "device": null,
+              "user": {
+                "id": "52bd2d9c-2d89-416f-96c4-ca94245e22c8",
+                "displayName": null,
+                "tenantId": "ad4f158a-97c7-4914-a9bd-038ecde40ff3",
+                "identityProvider": "AAD"
+              }
+            }
+          }
+        ]
+      },
+      "lobbyBypassSettings": {
+        "scope": "unknownFutureValue",
+        "isDialInBypassEnabled": false
+      },
+      "joinMeetingIdSettings": {
+        "isPasscodeRequired": true,
+        "joinMeetingId": "12345",
+        "passcode": "Z3GYtQ"
+      },
+      "chatInfo": {
+        "threadId": "abc",
+        "messageId": "0",
+        "replyChainMessageId": null
+      },
+      "joinInformation": {
+        "content": "textContent",
+        "contentType": "html"
+      },
+      "watermarkProtection": {
+        "isEnabledForContentSharing": false,
+        "isEnabledForVideo": false
+      }
+    }
+  ]
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  endDateTime        : 2023-10-05T13:57:09.1889486Z
+  joinUrl            : https://teams.microsoft.com/l/meetup-join/abc
+  recordAutomatically: false
+  startDateTime      : 2023-10-05T12:57:09.1889486Z
+  subject            : null
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  @odata.context,id,creationDateTime,startDateTime,endDateTime,joinUrl,joinWebUrl,meetingCode,isBroadcast,autoAdmittedUsers,isEntryExitAnnounced,allowedPresenters,allowMeetingChat,shareMeetingChatHistoryDefault,allowTeamworkReactions,allowAttendeeToEnableMic,allowAttendeeToEnableCamera,recordAutomatically,allowParticipantsToChangeName,allowRecording,allowTranscription
```

Comment:
I see a column odata.context here, it should not be here, maybe you're using odata call with metadata..

Reply:
Done